common/scylla_configure.py

```diff
@@ -50,14 +48,22 @@ class ScyllaAmiConfigurator:
         'start_scylla_on_first_boot': True,
     }
 
-    INSTANCE_METADATA_URL = "http://169.254.169.254/latest/"
     DISABLE_START_FILE_PATH = Path("/etc/scylla/ami_disabled")
 
     def __init__(self, scylla_yaml_path="/etc/scylla/scylla.yaml"):
         self.scylla_yaml_path = Path(scylla_yaml_path)
         self.scylla_yaml_example_path = Path(scylla_yaml_path + ".example")
         self._scylla_yaml = {}
         self._instance_user_data = None
+        self._cloud_instance = None
+
+    @property
+    def cloud_instance(self):
+        if not self._cloud_instance:
+            sys.path.append('/opt/scylladb/scripts')
+            from scylla_util import get_cloud_instance
+            self._cloud_instance = get_cloud_instance()
```

Comment:
No need to address in this pull request, but I would like to understand why cloud provider support is in `scylla.git`, not in this repository...

Reply:
I guess since `scylla_setup` still need to use this logic (and historically it was there), anyhow both repos need those classes to understand in which cloud they are, and get some basic info out of it.